### PR TITLE
Refinements in section 1

### PR DESCRIPTION
**Pull request by ypriverol**
I have made major refinements in section 1 about OSS, misconceptions, and misuse of the term in the field. Please let me know what you think. 

- [ ] Re-wrote **What does it mean for software to be "open source"?** including all the comments provided and clearly state points about what is OS to avoid more misconceptions.  
- [ ] I moved the table about How to start with open source project (introduced by @bittremieux ) to another section to make it more clear in this section the concept of OSS and in the other how to start with a project and how to make it successful. 
- [ ] I wrote the section: **Transparency promotes scientific rigor** in a more positive way as suggested by some authors. 
- [ ] Re-wrote in a more consistent way the section **Shared knowledge pushes the field forward** including the remarks from MM that are extremely valuable in that section from the Skyline experience.

### CODE REVIEW

File: manuscript.md [hunk 1]
```diff
@@ -37,104 +37,81 @@ We will also explore how other omics fields are dealing with OSS and FAIR softwa
...
-For example, in genomics, the Genome Analysis Toolkit (GATK - https://github.com/broadinstitute/gatk), is an Apache 2.0 licensed project with over a hundred maintainers, more than 4000 pull requests from the community, and more than 26,000 citations [20644199].
+In proteomics - and bioinformatics in general - multiple misconceptions exist about open/closed source software: 
 
-(GATK is maybe a bad example, it was not OSS for a very long time. Samtools is maybe better)
+- Closed-source software with "academic" licenses are equivalent to open-source. This is not true, as academic licenses only refer to free-for-academic-use, but the source code is not necessarily open, shareable or can be modified. Even the term "academic" is not well-defined, as it can refer to a wide range of institutions and organizations. To simplify this complexity, we can define OSS as any software that uses an OSI-approved license (https://opensource.org/licenses). 
+- Open-source means the code accessible. This is not true, Open-source software does not only mean that the code is openly available, but also that it is allowed to be modified and shared.
```
Comment (timosachsenberg) [inline manuscript.md]:
```suggestion
- Open-source means that only the code accessible. This is not true, open-source software does not only mean that the code is openly available, but also that it is allowed to be modified and shared.
```

File: manuscript.md [hunk 2]
```diff
@@ -37,104 +37,81 @@ We will also explore how other omics fields are dealing with OSS and FAIR softwa
...
 
-In proteomics, Percolator (https://github.com/percolator/percolator) is an open-source project with more than 700 citations and 20 contributors [17952086]; and a core tool for multiple open-source tools [], commercial software [], and pipelines [].
-The idea that open-source software is of lower quality is a myth that has been debunked by the success of many open-source projects in the field of computational proteomics and bioinformatics. [If it was debunked why does Proteomics still have this problem? Why do we write this manuscript? I guess you can demonstrate the success of Genomics and the healthy software ecosystem there - and contrast it with Proteomics or Metabolomics] [M: the community thinks academic proteomics software is of lower quality ...  https://pubs.acs.org/doi/10.1021/acs.jproteome.8b00015]
+In addition to all these misconceptions and complexity, journals mandate code availability as a publication requirement, which has prompted multiple bad practices from software developers and bioinformaticians aiming to futfill the journal's requirements.
```
Comment (timosachsenberg) [inline manuscript.md]:
```suggestion
In addition to all these misconceptions and complexity, many journals mandate code availability as part of publishing, which has prompted multiple bad practices from software developers and bioinformaticians aiming to fulfill these requirements. 
```

File: manuscript.md [hunk 3]
```diff
@@ -37,104 +37,81 @@ We will also explore how other omics fields are dealing with OSS and FAIR softwa
...
-A common misconception is that closed-source software with "academic" licenses are equivalent to open-source. [W: What do you understand under academic licenses? Do you mean that the code is open to academics, or just that academics can run the software for free / at reduced cost? This should be defined for the reader.] [B: usually this refers to free-for-academic-use only. And this is bullshit, because no-one can define academics.]
+- Researchers upload closed-source software to platforms like GitHub, giving an impression of openness with features such as issue tracking, while the actual source code remains inaccessible. Although often well-intentioned, this practice can mislead other scientists and, in our view, should be discouraged. 
+- Software is deposited in GitHub as open-source during the submission of the manuscript, but after publication; software licenses in GitHub repositories are changed over time, and repositories themselves can be deleted or made private, complicating efforts to ensure long-term accessibility and reliability.
```
Comment (timosachsenberg) [inline manuscript.md]:
```suggestion
- Software is deposited in GitHub as open-source during the submission of the manuscript, but after publication; software licenses in GitHub repositories are changed, repositories deleted or made private, complicating efforts to ensure long-term accessibility.
```

File: manuscript.md [hunk 4]
```diff
@@ -37,104 +37,81 @@ We will also explore how other omics fields are dealing with OSS and FAIR softwa
...
 
-[B: Rant about "free-for-academic-use": There is no defintion of academics, is an institute payed 50% from a state, 50% by industry, academic or not? Is an industry partner that has a public grant together with some university "academic" or not?
-Even worse, if I'm a packager, creating conda packages or containers, who is responsible to check that the consumer of those packages is academic? If I run a webserver with those tools, who is responsible to check the "academic"? We could shorten this section and simply state OSS is only software that adhers to one of the OSI approved licenses: https://opensource.org/licenses]
+Knowing all of these misconceptions and complexity, we want to make clear for the community, users, developers, readers of manuscripts, and reviewers what is open-source in proteomics, what is not, and how to make it clear for everyone:
```
Comment (timosachsenberg) [inline manuscript.md]:
```suggestion
Given the many misconceptions and complexities surrounding open-source in proteomics, we aim to clarify for the entire community - users, developers, manuscript readers, and reviewers - what constitutes open-source, what does not, and how to transparently communicate the state of own software: 
```

File: manuscript.md [hunk 5]
```diff
@@ -37,104 +37,81 @@ We will also explore how other omics fields are dealing with OSS and FAIR softwa
...
 
-[W: I moved this paragraph down because I think it makes more sense to first discuss different aspects of OSS, and afterwards transition to closed source.]
-In contrast, closed source means that the source code of the software is not published and it is not shared with the public for anyone to look at or change. [True, but it can also be public, but not allowed to be modified or shared.]
-A common misconception is that closed-source software with "academic" licenses are equivalent to open-source. [W: What do you understand under academic licenses? Do you mean that the code is open to academics, or just that academics can run the software for free / at reduced cost? This should be defined for the reader.] [B: usually this refers to free-for-academic-use only. And this is bullshit, because no-one can define academics.]
+- Researchers upload closed-source software to platforms like GitHub, giving an impression of openness with features such as issue tracking, while the actual source code remains inaccessible. Although often well-intentioned, this practice can mislead other scientists and, in our view, should be discouraged.
```
Comment (RobbinBouwmeester) [inline manuscript.md:54]: Although not against the terms of GH. I think the sentence as-is is fine.

File: manuscript.md (hunk 3 above)
Comment (RobbinBouwmeester) [reply manuscript.md]: Maybe something like this?

"Software deposited on GitHub as open-source during the submission can change its license after publication of the manuscript, or even worse, repositories get deleted or made private. These steps complicate efforts to ensure long-term accessibility to the code."

File: manuscript.md [hunk 6]
```diff
@@ -37,104 +37,81 @@ We will also explore how other omics fields are dealing with OSS and FAIR softwa
...
-| 9. **Monitor and measure success.** Track metrics like repository stars, downloads, citations, or code contributions to gauge adoption and impact. Regularly collect user feedback and address concerns or feature requests to ensure the project stays relevant and useful to its audience. |
-[W: Open to discussion. Please carefully double-check whether these steps make sense and that I haven't forgotten anything.] [B: 7-9 is often not possible in our current funding system - we should make clear that the first bullet points are a must have and the last 3 are a goal][M: A major problem is that there are very very few examples of this in proteomics ... there are a handful tops. I think this needs to be toned down a lot. This should be a goal but the funding agencies have to be supportive of this first. The issue is that the NIH doesn't want to fund open source software development and they definitely don't want to fund repositories.
-]
+- **Freedom to Modify and Distribute**: Open-source software allows users not only to access the code but also to modify it and share these modifications, encouraging innovation and collaboration.
```
Comment (RobbinBouwmeester) [inline manuscript.md]: Could clarify some confusion about open source software vs source-available but no modifications can be me (non-OSI):

"Open-source software, in contrast to source-available software, allows users ..."

File: manuscript.md (hunk 3 above)
Comment (bittremieux) [reply manuscript.md]: This is a pretty complex sentence, I suggest splitting it up in two.

File: manuscript.md [hunk 7]
```diff
@@ -37,104 +37,81 @@ We will also explore how other omics fields are dealing with OSS and FAIR softwa
...
+Open-source software (OSS) fosters a collaborative ecosystem where researchers across institutions can freely contribute, refine, and extend tools, accelerating scientific progress. 
+Unlike proprietary software that confines advancements to specific labs or companies, OSS allows researchers to build on each other’s work without duplicating efforts, promoting efficient resource use and transforming individual achievements into collective gains. 
+By democratizing access to high-quality tools, OSS enables scientists from resource-limited settings to leverage the same technology as those in well-funded institutions, fostering a diverse global community that drives innovation forward.
 
+Proteomics has greatly benefited from this open-source approach.
```
Comment (bittremieux) [inline manuscript.md:97]: This section is very nice now. 👍

**Review by bittremieux — APPROVED** [review]

**Review by RobbinBouwmeester — APPROVED** [review]

**Pull request merged**